src/hooks/useUser/useUser.tsx

```diff
@@ -61,6 +67,21 @@ export const ActiveUserProvider: React.FC = ({ children }) => {
     throw activeMembershipError
   }
 
+  const checkIfPolkadotExtensionInstalled = useCallback(async () => {
+    const res = await fetch(EXTENSION_URL)
+    const polkaDotExtensionInstalled = res.ok
+    if (polkaDotExtensionInstalled) {
+      console.warn('Polkadot extension disabled')
+      openDialog({
```

Comment:
I'm not sure if this should open for user right away. If I visit studio for the first time and have the extension installed, I will get a prompt to enable access from the extension. Once I enable it, I will see this dialog even though I just allowed the access. That's a bad UX. What if instead we show it only in `Add Polkadot extension` step?

Reply:
Maybe we should just skip the detection entirely and just show a link to this in the mentioned step in sign in? I'm not sure what additional value this brings us

Reply:
> Maybe we should just skip the detection entirely and just show a link to this in the mentioned step in sign in? I'm not sure what additional value this brings us

yeah, that seems to make most sense at the moment. Removed detection - currently dialog with instruction will appear on click in the sign in step